test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/DefaultTagHelperContentTest.cs

```diff
@@ -87,6 +87,65 @@ public void CanAppendContent()
         }
 
         [Fact]
+        public void CanAppendFormatContent()
+        {
+            // Arrange
+            var tagHelperContent = new DefaultTagHelperContent();
+            var expected = "Hello World!";
+
+            // Act
+            tagHelperContent.AppendFormat("{0} World!", "Hello");
+
+            // Assert
+            Assert.Equal(expected, tagHelperContent.GetContent());
+        }
+
+        [Fact]
+        public void CanAppendFormat_WithCulture()
+        {
+            // Arrange
+            var tagHelperContent = new DefaultTagHelperContent();
+            var expected = "Hello World!";
+
+            // Act
+            tagHelperContent.AppendFormat(CultureInfo.InvariantCulture, "{0} World!", "Hello");
+
+            // Assert
+            Assert.Equal(expected, tagHelperContent.GetContent());
+        }
+
+        [Fact]
```

Comment:
Add a test with `[ReplaceCulture("fr-FR")]` to verify we pick the default culture.